File: src/gui_app/conexao.py
import pymysql

DB_CONFIG = {
    "host": "localhost",
    "user": "root",
    "password": "",
    "database": "db_provapython",
    "cursorclass": pymysql.cursors.DictCursor
}

def conectar_bd():
    try:
        conn = pymysql.connect(**DB_CONFIG)
        return conn
    except Exception as e:
        print("Erro ao conectar ao banco:", e)
        return None

File: src/web_app/sys_rast_log/conexao.php
<?php
$servername = "localhost";
$username   = "root";
$password   = "";
$database   = "db_provapython";

$conn = new mysqli($servername, $username, $password, $database);

if ($conn->connect_error) {
    die("Erro na conexão: " . $conn->connect_error);
}

$conn->set_charset("utf8");
?>
